skills/csharp/pointofinterestskill/Dialogs/MainDialog.cs

```diff
@@ -26,7 +26,7 @@ namespace PointOfInterestSkill.Dialogs
     public class MainDialog : ComponentDialog
     {
         private BotServices _services;
-        private LocaleTemplateManager _responseManager;
+        private LocaleTemplateManager templateManager;
```

Comment:
Should we unify the naming style, using underline or not?

Reply:
Oops, slipped!